### ISSUE TITLE
[terra-hyperlink] Axe Violation due to color contrast of disabled hyperlinks

### ISSUE THREAD

**Issue by saket2403** [cerner/terra-core#2725]
## Description
The axe plugin gives a error due to less color contrast for disabled hyperlinks

## Error Report
```
Element has insufficient color contrast of 1.51 (foreground color: #d2d2d3, background color: #ffffff, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1
```
## Possible Solution
Adding `aria-disabled` for disabled hyperlinks
